### PR TITLE
Gexrgia/login

### PR DESCRIPTION
**Pull request by gexrgia**
Login and Registration

### CODE REVIEW

File: static/login.js [hunk 1]
```diff
@@ -1,4 +1,65 @@
...
+      'Content-Type': 'application/json',
+      'Accept': 'application/json'
+    },
+    body: JSON.stringify({
+      username: user_input, 
+      password: pass_input
+    })
+  })
+  
+  .then(res => res.json)
```
Comment (VijayStroup) [inline static/login.js:22]: res.json()

File: static/login.js [hunk 2]
```diff
@@ -1,4 +1,65 @@
...
+  .then(data => console.log(data['token']))
+  .then(res => {
+    if (res.ok) {
+      setCookie('token', data['token'], 30);
```
Comment (VijayStroup) [inline static/login.js:26]: You want to set the cookie when u recieve the json and its good.
I would do something like this:
```js
.then(res => {
  if (res.ok) {
    res.json().then(data => setCookie(...))
  } else {
    console.log('Incorrect username/password')
  }
})

**Review by VijayStroup — CHANGES_REQUESTED** [review]

File: static/login.js (hunk 2 above)
Comment (gexrgia) [reply static/login.js:26]: Should I still include line 22 with this update?

Comment (VijayStroup) [reply static/login.js:26]: You can remove line 22  and 23 and replace the .then starting on line 24 with the code block I sent.

**Pull request merged**